apps/desktop-wallet/src/features/assetsLists/AssetsBadgesList.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { AddressHash } from '@alephium/shared'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { useAddressesTokensBalances } from '@/api/addressesBalancesDataHooks'
+import { useAddressesNfts } from '@/api/addressesNftsDataHooks'
+import { useAddressesUnlistedNonStandardTokenIds } from '@/api/addressesUnlistedTokensHooks'
+import AssetBadge, { AssetBadgeStyleProps } from '@/components/AssetBadge'
+import SkeletonLoader from '@/components/SkeletonLoader'
+import useAddressesSortedFungibleTokens from '@/features/assetsLists/useSortedFungibleTokens'
+
+interface AssetsLogosListProps extends AssetBadgeStyleProps {
+  addressHash: AddressHash
+  maxDisplayedAssets?: number
+  showAmount?: boolean
+}
+
+const AssetsLogosList = ({
+  addressHash,
+  maxDisplayedAssets = 8,
+  className,
+  showAmount,
+  ...badgeProps
+}: AssetsLogosListProps) => {
+  const { t } = useTranslation()
+  const { data: tokensBalances } = useAddressesTokensBalances(addressHash)
+  const { data: sortedFTs, isLoading: isLoadingSortedFTs } = useAddressesSortedFungibleTokens(addressHash)
+  const { data: NFTs, isLoading: isLoadingNFTs } = useAddressesNfts(addressHash)
+  const { data: nonStandardTokens, isLoading: isLoadingNonStandardTokens } =
+    useAddressesUnlistedNonStandardTokenIds(addressHash)
+
+  const displayedFTs = sortedFTs.slice(0, maxDisplayedAssets)
```

Comment:
After using it, what do you think about having `FTs` and `NFTs`?
I'm not yet convinced, I find it easy to confuse variables. Maybe I just need to get used to it.

Reply:
Funny, I just sent a msg on slack to discuss this, without seeing this message 😆  🧠 ⚡ 🧠  Let's continue the discussion there!

Reply:
So, I am in favor of the list
- `listedFT`
- `unlistedFT`
- `NFT`
- `NST`

I wouldn't go as far as to name the first 2 `LFT` and `UFT`. WDYT?

Reply:
btw, the following open PR implements the above proposal, you can check it out there to get a feeling for it: https://github.com/alephium/alephium-frontend/pull/791